test/sai_test/sai_tunnel_test.py

```diff
@@ -241,9 +456,9 @@ def runTest(self):
         finally:
             pass
 
-class IPInIPTunnelEncapv6Inv4Test(IpInIpTnnnelBase):
+class SviIPInIPTunnelEncapv6Inv4Test(IpInIpTnnnelBase):
```

Comment:
@zhiyuan0112 Please check the case name in our statistics data
SviIPInIPTunnelEncapv6Inv4Test
SviIPInIPTunnelEncapv6Inv4Test